dali/test/test_tensors.h

```diff
@@ -55,7 +55,7 @@ class TestTensorList {
       cpumem_ = mm::alloc_raw_unique<char, mm::memory_kind::host>(size, 256);
       if (gpumem_)
         CUDA_CALL(cudaMemcpyAsync(cpumem_.get(), gpumem_.get(), size,
-                                  cudaMemcpyDeviceToHost, stream));
+                                  cudaMemcpyDefault, stream));
```

Comment:
This will slow thing down - is it _really_ needed? After all, we _know_ that it's a D2H copy.

Reply:
I checked [cudaMemcpy docs](https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__MEMORY.html#group__CUDART__MEMORY_1gc263dbe6574220cc776b45438fc351e8) as well as [cudaMemcpyDefault definition](https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__TYPES.html#group__CUDART__TYPES_1g18fa99055ee694244a270e4d5101e95b) and there's no mention of higher resource usage, only that it uses UVA. Do you happen to have any reference for this?

Also, I believe CUDA engineers point out, that there are virtually no difference between these.

Reply:
Reverted